lib/ws.c

```diff
@@ -175,9 +187,16 @@ static CURLcode ws_dec_read_head(struct ws_decoder *dec,
       dec->head[0] = *inbuf;
       Curl_bufq_skip(inraw, 1);
 
+      if(!ws_frame_rsv_supported(dec, dec->head[0])) {
+        failf(data, "WS: unknown rsv: %x", dec->head[0] & WSBIT_RSV_MASK);
```

Comment:
Maybe elaborate a tad bit more in the error message so that a user has a better chance to understand what it means?